v20.1/export.md

```diff
@@ -36,19 +36,19 @@ Only members of the `admin` role can run `EXPORT`. By default, the `root` user b
 
  Parameter | Description
 -----------|-------------
- `file_location` | Specify the URL of the file location where you want to store the exported CSV data.
+ `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported CSV data.<br><br>Note: Exports do not generate unique names, so each export should have a unique destination to avoid overwriting. 
```

Comment:
`Exports do not generate unique names` isn't clear to me -- since it does create unique file names *within* the export, just not *across* exports.

Otherwise lgtm